content/en/tracing/metrics/runtime_metrics/data_collected.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Runtime Metrics Data Collected
+type: multi-code-lang
+---
+
+This page lists data collected by the APM runtime metrics.
+
+## Java
+
+{{< get-metrics-from-git "java" >}}
```

Comment:
Not sure how to update this, but it seems like the Java one is missing [jvm.loaded_classes](https://app.datadoghq.com/dash/integration/256/jvm-metrics?fromUser=false&fullscreen_end_ts=1741213859245&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=overview&fullscreen_start_ts=1741210259245&fullscreen_widget=4428489373433180&refresh_mode=sliding&from_ts=1741210233187&to_ts=1741213833187&live=true)

Reply:
Added via https://github.com/DataDog/dogweb/pull/137966